src/components/ResoucesList.tsx

```diff
@@ -0,0 +1,242 @@
+            onChange={(e) => setFilterTags(handleTagsFilter(e.target.value))}
+            required
+            className="resourceTag-Week6"
+          ></input>
+          <label>Week7</label>
+          <input
+            value="Week7"
+            type="checkbox"
+            name="resource_tags"
+            onChange={(e) => setFilterTags(handleTagsFilter(e.target.value))}
+            required
+            className="resourceTag-Week7"
+          ></input>
+          <label>Week8</label>
+          <input
+            value="Week8"
+            type="checkbox"
+            name="resource_tags"
+            onChange={(e) => setFilterTags(handleTagsFilter(e.target.value))}
+            required
+            className="resourceTag-Week8"
+          ></input>
+        </div>
+        <div className="searchBar">
+          <input
+            className="search-input"
+            value={searchValue}
+            placeholder="search for a resource..."
+            onChange={(e) => setSearchValue(e.target.value)}
+          ></input>
+        </div>
+      </div>
+      {resources.length ? (
+        <div className="resource-container">
+          {resources
+            .filter((resource) =>
+              resource.resource_name
+                .toLocaleLowerCase()
+                .includes(searchValue.toLocaleLowerCase())
+            )
+            .filter((resource) => resource.resource_tags.includes(filterTags))
+            .map((resource) => {
+              return (
+                <div className="resource-item" key={resource.resource_id}>
+                  <div className="resourceSummary">
+                    <h3 className="title">{resource.resource_name}</h3>
+                    <div className="resource-info">
+                      <button
+                        onClick={() => handleLikeResource(resource.resource_id)}
+                      >
+                        {resource.resource_likes}👍
+                      </button>
+                      <button>{resource.resource_dislikes}👎</button>
+                      <p>
+                        {format(
+                          Date.parse(resource.resource_post_date.toString()),
+                          "dd/MM/yy"
+                        )}
+                      </p>
+
+                      <p className="authorName">{resource.author_name}</p>
+                      <p className="description">
+                        {textSummary(resource.resource_description, 50)}
+                      </p>
+                      <p>{resource.resource_tags}</p>
+
+                      <Link
+                        className="resourceButton"
+                        to={`/fullresource/${resource.resource_id}`}
+                      >
+                        See Full Resource
+                      </Link>
+                    </div>
+                  </div>
+                </div>
+              );
+            })}
+        </div>
+      ) : (
+        <h1>Loading Data...</h1>
```

Comment:
Nice to have a placeholder while the data is loading!!!

Reply:
thank you!